contracts/StakedCelo.sol

```diff
@@ -46,4 +67,73 @@ contract StakedCelo is ERC20Upgradeable, UUPSOwnableUpgradeable, Managed {
     function burn(address from, uint256 amount) external onlyManager {
         _burn(from, amount);
     }
+
+    /**
+     * @notice Locks stCELO from an address.
+     * @param account The address that will have its stCELO balance locked.
+     * @param amount The amount of stCELO to lock.
+     */
+    function lockBalance(address account, uint256 amount) external onlyManager {
+        uint256 lockedBalance = _lockedBalances[account];
+        if (lockedBalance < amount) {
```

Comment:
If there is a scenario 

>  user votes on proposal 1 with 100stCelo -> 100stCelo is locked

> user votes on proposal 2 with 50stCelo -> since already 100stCelo is locked we don't have to lock anything